pylint/lint/parallel.py

```diff
@@ -130,7 +130,7 @@ def check_parallel(
     """Use the given linter to lint the files with given amount of workers (jobs).
 
     This splits the work filestream-by-filestream. If you need to do work across
-    multiple files, as in the similarity-checker, then implement the map/reduce mixin functionality.
```

Comment:
is it no longer [considered] a mix-in class?

Reply:
Right, see #6383 and #8409

Reply:
Oooooh, hadn't seen those, sexy